### ISSUE TITLE
No default value for select fields

### ISSUE THREAD

**Issue by cristianuibar** [Laravel-Backpack/CRUD#1401]
# Bug report

### What I did:
I added a `default` value to my select2 field

### What I expected to happen:
The default value should be checked but ignored if the old input is present.

### What happened:
Nothing. The field continues to work and ignore my default value.

### What I've already tried to fix it:
I fixed it by creating my own and overwrite the `select2.blade.php` file and add an extra check for the default value on line `18` like so:
```php
                @if(old($field['name']) == $connected_entity_entry->getKey() || (is_null(old($field['name'])) && isset($field['value']) && $field['value'] == $connected_entity_entry->getKey()) || 
                (is_null(old($field['name'])) && !isset($field['value']) && isset($field['default']) && $field['default'] == $connected_entity_entry->getKey()) ? $field['default'] : false)
                    <option value="{{ $connected_entity_entry->getKey() }}" selected>{{ $connected_entity_entry->{$field['attribute']} }}</option>
                @else
                    <option value="{{ $connected_entity_entry->getKey() }}">{{ $connected_entity_entry->{$field['attribute']} }}</option>
                @endif
```

I'm not sure if it's the perfect method of doing it.

### Backpack, Laravel, PHP, DB version:
BP: 3.4
LR: 5.6
PHP: 7.2
DB: MariaDB 10.1

**Comment by tabacitu** [comment]
You're totally right. Thank you @cristianuibar - seems the ```default``` attribute has been removed by mistake. I've just added it, it'll be available with ```composer update``` after it's bundled with a few more changes and tagged (later today or in 1-2 days). It'll be available in version 3.4.11.

Thanks a lot, cheers!
